rollup.config.js

```diff
@@ -0,0 +1,32 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import {terser} from 'rollup-plugin-terser';
+import nodeResolve from '@rollup/plugin-node-resolve';
+
+const extensions = ['.js', '.ts'];
+
+export default {
+  input: 'src/index.ts',
+  output: {
+    file: 'lib/index.iife.min.js',
+    format: 'iife',
+    name: 'walletStandardBrave',
+    sourcemap: true,
+    plugins: [terser({mangle: false, compress: false})],
+  },
+  plugins: [
+    commonjs(),
+    nodeResolve({
+      browser: true,
+      dedupe: ['bn.js', 'buffer'],
+      extensions,
+      preferBuiltins: false,
+    }),
+    babel({
```

Comment:
I'm not sure it's best practice to use babel these days - seems like we can just use typescript?

Reply:
without it we will have error like this
```
src/index.ts → lib/index.iife.min.js...
[!] Error: Unexpected token (Note that you need plugins to import files that are not JavaScript)
src/initialize.ts (3:12)
1: import { registerWallet } from './register.js';
2: import { BraveWalletWallet } from './wallet.js';
3: import type { BraveWallet } from './window.js';
               ^
4:
5: export function initialize(braveWallet: BraveWallet): void {
Error: Unexpected token (Note that you need plugins to import files that are not JavaScript)
    at error (/Users/darkdh/Projects/wallet-standard-brave/node_modules/rollup/dist/shared/rollup.js:198:30)
    at Module.error (/Users/darkdh/Projects/wallet-standard-brave/node_modules/rollup/dist/shared/rollup.js:12521:16)
    at Module.tryParse (/Users/darkdh/Projects/wallet-standard-brave/node_modules/rollup/dist/shared/rollup.js:12897:25)
    at Module.setSource (/Users/darkdh/Projects/wallet-standard-brave/node_modules/rollup/dist/shared/rollup.js:12800:24)
    at ModuleLoader.addModuleSource (/Users/darkdh/Projects/wallet-standard-brave/node_modules/rollup/dist/shared/rollup.js:22264:20)
```

Reply:
both config files are copied from https://github.com/solana-labs/solana-web3.js and simplified for our need

Reply:
oh I didn't know this was partially forked. Could you maybe briefly try using the typescript rollup plugin instead of the babel plugin as it looks like babel is only used to provide typescript conversion? If that doesn't immediately work then I'd say you're fine proceed with babel. It's just preferred not to because it can add complexity to the output and bundling process (another thing that's hard to diagnose when you have resolve or type issues!)

Reply:
I tried using `@rollup/plugin-typescript` but it will have this kind of runtime error
<img width="687" alt="Screen Shot 2023-02-06 at 11 07 32" src="https://user-images.githubusercontent.com/11330831/217062460-e0c9fb89-4144-45d2-a146-f50619386ed2.png">
I think we will have to create an issue for that and come back to it later.

Reply:
https://github.com/brave/wallet-standard-brave/issues/12